cmd/program/program.go

```diff
@@ -866,35 +881,64 @@ func (p *Project) CreateViteReactProject(projectPath string) error {
 			return fmt.Errorf("failed to initialize Tailwind: %w", err)
 		}
 
-		// use the tailwind config file
-		err = os.WriteFile("tailwind.config.js", advanced.ReactTailwindConfigTemplate(), 0644)
-		if err != nil {
-			return fmt.Errorf("failed to write tailwind config: %w", err)
-		}
-
 		srcDir := filepath.Join(frontendPath, "src")
 		if err := os.MkdirAll(srcDir, 0755); err != nil {
 			return fmt.Errorf("failed to create src directory: %w", err)
 		}
 
-		err = os.WriteFile(filepath.Join(srcDir, "index.css"), advanced.InputCssTemplateReact(), 0644)
-		if err != nil {
-			return fmt.Errorf("failed to update index.css: %w", err)
-		}
+		if p.AdvancedOptions[string(flags.React)] {
+			// use the tailwind config file for React
+			err = os.WriteFile("tailwind.config.js", advanced.ReactTailwindConfigTemplate(), 0644)
+			if err != nil {
+				return fmt.Errorf("failed to write tailwind config: %w", err)
+			}
+			// update index.css for React
+			err = os.WriteFile(filepath.Join(srcDir, "index.css"), advanced.InputCssTemplateReact(), 0644)
+			if err != nil {
+				return fmt.Errorf("failed to update index.css: %w", err)
+			}
 
-		if err := os.WriteFile(filepath.Join(srcDir, "App.tsx"), advanced.ReactTailwindAppfile(), 0644); err != nil {
-			return fmt.Errorf("failed to write App.tsx template: %w", err)
-		}
+			if err := os.WriteFile(filepath.Join(srcDir, "App.tsx"), advanced.ReactTailwindAppfile(), 0644); err != nil {
+				return fmt.Errorf("failed to write App.tsx template: %w", err)
+			}
 
-		if err := os.Remove(filepath.Join(srcDir, "App.css")); err != nil {
-			// Don't return error if file doesn't exist
-			if !os.IsNotExist(err) {
-				return fmt.Errorf("failed to remove App.css: %w", err)
+			// this only exists in the React template
+			if err := os.Remove(filepath.Join(srcDir, "App.css")); err != nil {
+				// Don't return error if file doesn't exist
+				if !os.IsNotExist(err) {
+					return fmt.Errorf("failed to remove App.css: %w", err)
+				}
+			}
+		} else if p.AdvancedOptions[string(flags.Svelte)] {
+			// use the tailwind config file for svelte
+			err = os.WriteFile("tailwind.config.js", advanced.SvelteTailwindConfigTemplate(), 0644)
+			if err != nil {
+				return fmt.Errorf("failed to write tailwind config: %w", err)
+			}
+			// update app.css for Svelte
+			err = os.WriteFile(filepath.Join(srcDir, "app.css"), advanced.InputCssTemplateSvelte(), 0644)
+			if err != nil {
+				return fmt.Errorf("failed to update app.css: %w", err)
+			}
+
+			if err := os.WriteFile(filepath.Join(srcDir, "App.svelte"), advanced.SvelteTailwindAppfile(), 0644); err != nil {
+				return fmt.Errorf("failed to write App.svelte template: %w", err)
 			}
 		}
 
 		// set to false to not re-do in next step
 		p.AdvancedOptions[string(flags.Tailwind)] = false
+	} else {
```

Comment:
- moved the copying of template file in else block of tailwind as currently the app file was written once and then again it was overwritten if the tailwind one needed to be copied so now it is only written once.